app/Http/Controllers/Api/ProjectLabelTreeController.php

```diff
@@ -61,7 +61,7 @@ public function index($id)
     /**
      * Display label trees that match the given name and can be used by the specified project.
      *
-     * @api {get} projects/:id/label-trees/available Get all available label trees
+     * @api {get} projects/:id/label-trees/available Get matching label trees
```

Comment:
The name is still missing:

```suggestion
     * @api {get} projects/:id/label-trees/available/:name Get matching label trees
```